pycds/orm/tables.py

```diff
@@ -297,6 +298,9 @@ class Variable(Base):
         UniqueConstraint(
             "network_id", "net_var_name", name="network_variable_name_unique"
         ),
+        CheckConstraint(
+            "net_var_name !~ '\W'", name="ck_net_var_name_valid_identifier"
```

Comment:
Containing whitespace isn't the only thing that would disqualify a string as functioning as an SQL identifier... in fact the `\W` regex refers to "non-word characters".

Perhaps we should consider aligning this constraint more closely with the [definition of an SQL identifier](https://www.postgresql.org/docs/current/sql-syntax-lexical.html#SQL-SYNTAX-IDENTIFIERS)? Specifically:

> SQL identifiers and key words must begin with a letter (a-z, but also letters with diacritical marks and non-Latin letters) or an underscore (_). Subsequent characters in an identifier or key word can be letters, underscores, digits (0-9), or dollar signs ($).

We might be able to use [`parse_ident()`](https://www.postgresql.org/docs/current/functions-string.html) or another function to accomplish this.

Reply:
Well spotted. The definition of `\w` is quite variable; in Python apparently it captures a wide range of "letter" characters from multiple languages. In other implementations, it is very restrictive.

Reply:
I've made adjustments to the migration and tests based on these observations.